integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -481,6 +481,19 @@ fuse:
 #   logserver: # Used by the logserver container
 #     alluxio-hdfs-config: hdfsConfig
 
+
+##  ConfigMaps ##
+
+# Format: (<name>:<mount path under /configmaps/>):
+# configmaps:
+#   master: # Shared by master and jobMaster containers
+#     alluxio-krb5-config: krb5Config
+#   worker: # Shared by worker and jobWorker containers
+#     alluxio-krb5-config: krb5Config
+#   logserver: # Used by the logserver container
+#     alluxio-krb5-config: krb5Config
```

Comment:
```suggestion
#   master: # Shared by master and jobMaster containers
#     alluxio-hdfs-config: hdfsConfig
#   worker: # Shared by worker and jobWorker containers
#     alluxio-hdfs-config: hdfsConfig
#   logserver: # Used by the logserver container
#     alluxio-hdfs-config: hdfsConfig
```

Reply:
made changes as per the above comment @jiacheliu3